.github/workflows/ci.yml

```diff
@@ -69,9 +104,15 @@ jobs:
         with:
           path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
           key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
-      - name: Build
+      - name: Install
+        run: |
+          BABEL_ENV=test-legacy make -j bootstrap-only
+      - uses: actions/download-artifact@v2
+        with:
+          name: babel-artifact
+      - name: Generate runtime helpers
         run: |
-          BABEL_ENV=test-legacy make -j bootstrap
+          make build-plugin-transform-runtime-dist
```

Comment:
This step is moved to test executor (instead of the Build job) because it is IO-intensive (1000 files), caching and restoring and slower than generating.